chainstate/src/lib.rs

```diff
@@ -77,5 +83,23 @@ pub fn make_chainstate<S: chainstate_storage::BlockchainStorage + 'static>(
     Ok(Box::new(cons_interface))
 }
 
+pub fn make_chainstate_no_genesis<S: chainstate_storage::BlockchainStorage + 'static>(
```

Comment:
Are you sure this is necessary? I don't think this is used. I followed the call tree and I don't see it used.

Reply:
It's used in the test framework, but only one test requires it. I'll move this test back to the chainstate crate and have it not use the framework, then we can get rid of this function.

Reply:
Done.